packages/botonic-plugin-contentful/src/nlp/normalizer.ts

```diff
@@ -145,13 +226,12 @@ export class Normalizer {
       const tokenStems = stemmer.stem([token])
       words = words.concat(tokenStems.map(stem => new Word(token, stem)))
     }
-    return new NormalizedUtterance(txt, words, numStopWords == tokens.length)
+    return new NormalizedUtterance(raw, words, numStopWords == tokens.length)
```

Comment:
Why passing `raw` instead of `txt` here?

Reply:
because the first argument of NormalizedUtterance expected a raw :-)